lib/src/main/java/com/smileidentity/SmileID.kt

```diff
@@ -135,6 +138,19 @@ object SmileID {
         api = retrofit.create(SmileIDService::class.java)
     }
 
+    /**
+     * The callback mechanism allows for asynchronous job requests and responses.
+     * While the job_status API can be polled to get a result, a better method is to set up a
+     * callback url and let the system POST a JSON response.
+     *
+     * @param callbackUrl The callback url that will be used to asynchronously send results of your
+     * job requests
+     */
+    @JvmStatic
+    fun setCallbackUrl(callbackUrl: String) {
```

Comment:
One point Japhet has made in the past is that partners sometimes pass in invalid URLs (i.e. missing `https://` scheme) which causes integration issues. Can we instead pass in a `URL` type here? and also make it optional, so that the callbackurl can be removed if necessary

```suggestion
    fun setCallbackUrl(callbackUrl: URL?) {
```

Reply:
i.e. https://github.com/smileidentity/ios/pull/81/commits/5f7cd812881f015b5f7673ce877b5d352a0c6207

Reply:
I had that change actually

let me make it nullable